gpu/gpu_darwin.go

```diff
@@ -66,3 +64,7 @@ func (l GpuInfoList) GetVisibleDevicesEnv() (string, string) {
 	// No-op on darwin
 	return "", ""
 }
+
+func LibraryDir() string {
```

Comment:
do we still need this if it's just returning a blank string?

Reply:
Only on Darwin which has no library dependencies for metal.  On Linux/Windows it returns a path we need to wire into `[LD_LIBRARY_]PATH` for the dependent libraries to resolve when we start the runner.  See https://github.com/ollama/ollama/blob/jmorganca/llama/gpu/gpu.go#L645

Reply:
Makes sense. I missed this since the compiler directives were under the fold.